include/session/config/user_groups.hpp

```diff
@@ -213,6 +218,12 @@ struct group_info : base_group_info {
     /// auth_data are empty.
     bool kicked() const;
 
+    /// Mark the group as permanently destroyed. This cannot be unset once set.
+    void destroyGroup();
+
+    /// Returns true if the group was destroyed by one of the admin.
+    bool isDestroyed() const;
```

Comment:
No method needed: `is_destroyed` is a public member.

Reply:
as `is_destroyed` should never be reset, would it make sense to make it private and keep the accessor method?

Reply:
As per other discussion, it sounds like we'll have some sort of state constant (rather than bool) for this, so the accessor makes sense as something like `bool is_destroyed() const { return state == DESTROYED; }`